### DISCUSSION TITLE
OpenTelemetryApi suggested version?

### DISCUSSION THREAD

**Discussion by manojburra719** [open-telemetry/opentelemetry-java-instrumentation#11207] [Q&A]
Hi, Currently we are using 1.16.0v for opentelemetry Java-instrumentation, TelemetryApi and openTelemetryExtensionApiversion libraries. We are planning to upgrade opentelemetry Java-instrumentation to 2.3.0 from 1.16.0.
What's the suggested version that we should use for TelemetryApi and openTelemetryExtensionApiversion libraries.? Java version we are using is 11.
Thanks

**Comment by laurit** [comment]
Instrumentation 2.3.0 is based on 1.37.0 api.

**Reply by manojburra719** [reply]
Thanks for your response. 
For Java 11, What's the recommended opentelemetry Java-instrumentation, TelemetryApi and openTelemetryExtensionApiversion(alpha) libraries that we should upgrade it for from 1.16.0v.??

**Reply by laurit** [reply]
we suggest using latest version
